docs/hooks/use-fetcher.md

```diff
@@ -61,7 +61,9 @@ fetcher.submit(formData);
 
 ## `fetcher.load(href)`
 
-Loads data from a route loader. While multiple nested routes can match a URL, only the leaf route will be called.
+Loads data from a route loader. While multiple nested routes can match a URL, only the leaf route will be called. 
+
+Note that when calling `load` on an index route's loader, you [must include](https://remix.run/docs/en/1.19.3/guides/routing#what-is-the-index-query-param) a `?index` query parameter in order to disambiguate between the `index.tsx` layout and the `root.tsx` route.
```

Comment:
> in order to disambiguate between the `index.tsx` layout and the `root.tsx` route.

I think the `root.tsx` is misleading here, `?index` is for every index route where there is need to disambiguate between a route and and _index route in its hierarchy, in RR and Remix `root` is usually used to refer to the root route, the first route that contains all other routes

```
- root.tsx <-- this is root
- routes/
  - posts.tsx <-- root layout for all routes beneath posts
  - posts._index.tsx <-- the index route beneath posts route
```

Also `_index.tsx` in this context isn't a layout per se, it is a leaf node among the route